palm/utils.py

```diff
@@ -36,14 +40,17 @@ def run_on_host(
     if major < 3 or minor < 5:
         raise UnsupportedVersion(f"Python version {major}.{minor} is not supported.")
     if minor < 7:
-        from subprocess import PIPE
-
-        kwargs.update(dict(stdout=PIPE, stderr=PIPE))
+        if not capture_output:
```

Comment:
wait so PIPE PIPE is the _opposite_ of capture output in the newer version of subprocess? I thought it was the equivalent?

Reply:
@jakeberesford-palmetto ^^ this is the one I'm talking about. as long as I got this backwards (It looks like I did) please merge away!

Reply:
wait, I think maybe I'm wrong actually. let me take another look at this